Health/Core/Views/CircleProgressView/CircleProgressView.swift

```diff
@@ -0,0 +1,257 @@
+        progressStackView.spacing = 2
+        progressStackView.distribution = .fill
+        
+        stepCountStackView.axis = .vertical
+        stepCountStackView.spacing = 6
+        stepCountStackView.distribution = .fill
+        
+        todayLabel.text = "오늘"
+        todayLabel.textColor = .systemMint
+        todayLabel.font = .systemFont(ofSize: 34, weight: .semibold)
+        todayLabel.textAlignment = .center
+        
+        percentageLabel.text = "86%"
+        percentageLabel.textColor = .label
+        percentageLabel.font = .systemFont(ofSize: 58, weight: .bold)
+        percentageLabel.textAlignment = .center
+        
+        stepTitleLabel.text = "걸음 수"
+        stepTitleLabel.textColor = .label
+        stepTitleLabel.font = .systemFont(ofSize: 28, weight: .medium)
+        stepTitleLabel.textAlignment = .center
+        
+        stepProgressLabel.text = "6,200 / 8,000"
+        stepProgressLabel.textColor = .label
+        stepProgressLabel.font = .systemFont(ofSize: 26, weight: .regular)
+        stepProgressLabel.textAlignment = .center
+    }
+    
+    override func setupConstraints() {
+        NSLayoutConstraint.activate([
+            containerStackView.centerXAnchor.constraint(equalTo: self.centerXAnchor),
+            containerStackView.centerYAnchor.constraint(equalTo: self.centerYAnchor)
+        ])
+    }
+
+    private func registerForTraitChanges() {
+        // 인터페이스 스타일(light/dark)이 변경될 때 레이아웃을 다시 그리도록 등록합니다.
+        registerForTraitChanges([UITraitUserInterfaceStyle.self]) { (self: Self, previousTraitCollection: UITraitCollection) in
+            self.setNeedsLayout()
+        }
+    }
+
+    private func updateContainerScale(_ scale: CGFloat) {
+        containerStackView.transform = CGAffineTransform(
+            scaleX: scale,
+            y: scale
+        )
+    }
+
+    private func drawStrokeCircle() {
+        let adjustedStartAngle = -90.radian
+        let calculatedEndAngle = ((currentValue / totalValue) * 360.0 - 90.0).radian
+        
+        drawStrokeCircle(
+            in: &backgroundGradientLayer,
+            startAngle: 0.radian,
+            endAngle: 360.radian,
+            lightStrokeColors: backgroundLightColors,
+            darkStrokeColors: backgroundDarkColors,
+            lineWidth: backgroundLineWidth
+        )
+        
+        drawStrokeCircle(
+            in: &foregroundGradientLayer,
+            startAngle: adjustedStartAngle,
+            endAngle: calculatedEndAngle,
+            lightStrokeColors: foregroundLightColors,
+            darkStrokeColors: foregroundDarkColors,
+            lineWidth: foregroundLineWidth,
+        )
+    }
+}
+
+extension CircleProgressView {
+
+    private func drawStrokeCircle(
+        in layer: inout CALayer?,
+        startAngle startRadian: Double,
+        endAngle endRadian: Double,
+        lightStrokeColors: [UIColor],
+        darkStrokeColors: [UIColor],
+        lineWidth: CGFloat,
```

Comment:
```suggestion
        lineWidth: CGFloat
```
여기 마지막 파라미터에 쉼표 제거하시면 빌드 될겁니다.